bentoml/keras.py

```diff
@@ -4,6 +4,10 @@
 from ._internal.frameworks.keras import load_model
 from ._internal.frameworks.keras import save_model
 from ._internal.frameworks.keras import get_runnable
+from ._internal.frameworks.keras import KerasOptions
+
+ModelOptions = KerasOptions
+
```

Comment:
```suggestion
from ._internal.frameworks.keras import KerasOptions as ModelOptions
```

Reply:
![Screenshot 2022-06-09 at 23 53 42](https://user-images.githubusercontent.com/29749331/172987052-879a803d-ad4d-4f5e-9d13-ebfe9bec14d1.png)

refactor(style): If we go to the above change, `ModelOptions` is under the module scope.

Reply:
Will update as suggested.